src/more/static.md

```diff
@@ -0,0 +1,42 @@
+# Static Content
+
+Static content in Yew is served from the `static` directory in your crate. For example, if you have a folder structure like:
```

Comment:
This is useful info, thanks for adding!

I think you're conflating Yew and `cargo-web` though. Yew doesn't do any serving or html generation. That's all `cargo-web` (or some other tool). Could you please clarify those points in this page?